addons/account_peppol/wizard/peppol_config_wizard.py

```diff
@@ -0,0 +1,143 @@
+from markupsafe import Markup
+
+from odoo import api, fields, models, _
+
+
+class PeppolService(models.TransientModel):
+    _name = 'account_peppol.service'
```

Comment:
Isn't this what you are supposed to remove? (from upgrade PR)

Reply:
Yes my bad, went a bit too fast :sweat: